doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2451 @@
+
+You can aggregate all the values in a matrix and you can aggregate them across columns or rows using the ``axis`` parameter:
+
+::
+  
+  data.max(axis=0)
+  data.max(axis=1)
+
+
+.. image:: images/np_matrix_aggregation_row.png
+
+Once you've created your matrices, you can add and multiply them using arithmetic operators if you have two matrices that are the same size.
+
+::
+
+  data + ones
+
+.. image:: images/np_matrix_arithmetic.png
+
+You can do these arithmetic operations on matrices of different sizes, but only if one matrix has only one column or one row. In this case, NumPy will use its broadcast rules for the operation.
+
+::
+
+  data + ones_row
+
+.. image:: images/np_matrix_broadcasting.png
+
+Be aware that when NumPy prints N-Dimensional arrays, the last axis is looped over the fastest while the first axis is the slowest. That means that:
+
+**Input:** 
+
+::
+
+  np.ones((4,3,2))
+
+Will print out like this:
+
+**Output:**
+
+::
+
+  array([[[1., 1.],
+        [1., 1.],
+        [1., 1.]],
+
+       [[1., 1.],
+        [1., 1.],
+        [1., 1.]],
+
+       [[1., 1.],
+        [1., 1.],
+        [1., 1.]],
+
+       [[1., 1.],
+        [1., 1.],
+        [1., 1.]]])
+
+ 
+There are often instances where we want NumPy to initialize the values of an array. NumPy offers methods like ``ones()``, ``zeros()``,  ``Random Generator``, and even the legacy method ``random.random()`` for these instances. All you need to do is pass in the number of elements you want it to generate.
+
+::
+
+  np.ones(3)
+  mp.zeros(3)
+  np.random.random(3)
+  
+.. image:: images/np_ones_zeros_random.png
+
+You can also use the ``ones()``, ``zeros()``, and ``random()`` methods to create an array if you give them a tuple describing the dimensions of the matrix.
+
+::
+
+  np.ones(3,2)
+  mp.zeros(3,2)
+  np.random.random(3,2)
+
+.. image:: images/np_ones_zeros_matrix.png
+
+Read more about initializing the values of an array with `ones here <https://docs.scipy.org/doc/numpy/reference/generated/numpy.ones.html>`_, `zeros here <https://docs.scipy.org/doc/numpy/reference/generated/numpy.zeros.html>`_, and `initializing empty arrays here <https://docs.scipy.org/doc/numpy/reference/generated/numpy.empty.html>`_.
+
+
+Generating random numbers
+-------------------------
+
+The use of random number generation is an important part of the configuration and evaluation of machine learning algorithms. Whether you need to randomly initialize weights in an artificial neural network, split data into random sets, or randomly shuffle your dataset, being able to generate random numbers (actually, repeatable pseudo-random numbers) is essential.
+
+You have a number of options when using NumPy for random number generation. Random Generator is NumPy's replacement for RandomState. The main difference between them is that Generator relies on an additional BitGenerator to manage state and generate the random bits, which are transformed into random values.
```

Comment:
Since users are new to NumPy here, they won't know what `RandomState` is yet. Hence better not to mention, it's probably more confusing than helpful

Reply:
some double backticks around code objects would be nice here

Reply:
Note 1: done, note 2: I think I took care of this, but after deleting RandomState, there was only "Random Generator" to double backtick, unless I'm not seeing what I'm supposed to see.

Reply:
"Generator.integers"...